codegen/templates/operator/main.tmpl

```diff
@@ -88,8 +94,13 @@ func main() {
     {{ end }}
 
     // Create our operator
-    op := operator.New()
-    op.AddController(controller)
+    runner := operator.New()
+    runner.AddController(controller)
```

Comment:
I figured that was a possibility, but I wanted to avoid unrelated breaking changes in this PR. We can definitely make those changes in a future PR, though.

Reply:
Sounds good! I'll mark this resolved for now, thank you!